ambari/metadata.csv

```diff
@@ -1,31 +1,31 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
-ambari.boottime,gauge,connection,millisecond,,Host boot time.,0,ambari,BootTime
```

Comment:
don't know why validate metadata does not catch that this is not a valid value

Reply:
Seems there is not validation about the `interval`: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py